client/components/reader-infinite-stream/style.scss

```diff
@@ -0,0 +1,7 @@
+.reader-infinite-stream__row-wrapper {
+	border-bottom: 1px solid lighten( $gray, 30% );
```

Comment:
yeah, my bad, side affect of the change to editorconfig we landed. https://github.com/Automattic/wp-calypso/pull/14839 so i can embarass myself slightly less often in the future.